### ISSUE TITLE
Loss going Nan or inf

### ISSUE THREAD

**Issue by KtK99** [cylcharles/Pytorch_exercise#1]
Hi,
I am working with a custom dataset. It is not too big, 17 images, with around 500 object instances in it. There is only one class. I have not made any changes to the code itself, except for writing the dataloader class for my data. If I get some response that would be very much appreciated. 
It seems that there are other similar threads elsewhere on this topic but they are all unresolved.
